x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -126,13 +126,15 @@ export function TimeseriesChart({
           onBrushEnd={(event) =>
             onBrushEnd({ x: (event as XYBrushEvent).x, history })
           }
-          theme={{
-            ...chartTheme,
-            areaSeriesStyle: {
-              line: { visible: false },
+          theme={[
+            customTheme,
+            {
+              areaSeriesStyle: {
+                line: { visible: false },
+              },
             },
-            ...customTheme,
-          }}
+            ...chartTheme,
+          ]}
```

Comment:
I would advise listing your themes in an array of partial themes like this, where the lower the index the higher the priority. I ran into issues where the `chartTheme` is imported from eui then spread over a new theme object manually which overrides entire property trees.

For example say you spread `chartTheme` and override `axes` with a single value.

```ts
theme={{
  ...chartTheme,
  axes: {
    tickLine: { 
      size: 5,
    },
  },
}}
```

instead of the remaining `axes` properties coming from `chartTheme` they come from the `baseTheme` assuming only the single `axes` override was to be defined.

When listed the themes in an array the charts logic acts much like javascript's object prototypal inherence, looking for the first theme in the array to have a given property, defaulting to the `baseTheme` value if not found.